articles/azure-sql/virtual-machines/windows/failover-cluster-instance-azure-shared-disks-manually-configure.md

```diff
@@ -30,13 +30,13 @@ To learn more, see an overview of [FCI with SQL Server on Azure VMs](failover-cl
 Before you complete the instructions in this article, you should already have:
 
 - An Azure subscription. Get started for [free](https://azure.microsoft.com/free/). 
-- [Two or more West Central US-prepared Windows Azure virtual machines](failover-cluster-instance-prepare-vm.md) in the same [availability set](../../../virtual-machines/linux/tutorial-availability-sets.md) and a [proximity placement group](../../../virtual-machines/windows/co-location.md#proximity-placement-groups), with the availability set that's created with Fault Domain and Update Domain set to **1**. 
+- [Two or more Windows Azure virtual machines](failover-cluster-instance-prepare-vm.md). Both [availability sets](../../../virtual-machines/windows/tutorial-availability-sets.md) and [proximity placement groups](../../../virtual-machines/windows/co-location.md#proximity-placement-groups) are supported. If PPG is utilized, all nodes must exist in the same group.
```

Comment:
Can we add "in a supported region" to the sentence here with a link to the regions where ASD on Premium SSD is supported? https://docs.microsoft.com/en-us/azure/virtual-machines/disks-shared-enable?tabs=azure-cli#regional-availability